src/Component/Best.jsx

```diff
@@ -1,14 +1,65 @@
-import React, {Component} from "react";
-
+import React, { Component } from "react";
+import axios from "axios";
+import moment from "moment";
+import CONSTANTS from "../Services/Hnapi";
```

Comment:
Hnapi? give the proper file name.

and like said in react the session, always use camelcase/smallcase for folder names